js/ui/appDisplay.js

```diff
@@ -206,8 +206,7 @@ const BaseAppView = new Lang.Class({
 
     _trimInvisible: function(items) {
         let appSystem = Shell.AppSystem.get_default();
-        return items.filter(Lang.bind(this,
-            function(itemId) {
+        return items.filter(Lang.bind(this, function(itemId) {
```

Comment:
This, and the rest of changes in this commit, should be squashed into their respective commits from both this PR and https://github.com/endlessm/gnome-shell/pull/165, instead of having a separate commit for these changes.